WooCommerce/Classes/POS/Presentation/Settings/PointOfSaleSettingsHardwareDetailView.swift

```diff
@@ -41,7 +48,13 @@ struct PointOfSaleSettingsHardwareDetailView: View {
                         }
                     }
                 }
+                .listRowSeparator(.hidden)
             }
+            .listStyle(.plain)
```

Comment:
In general I found very tricky to adapt Lists to the POS style, most likely when we implement the next iteration of designs we want to get rid of List entirely and build the view up from zero.